### ISSUE TITLE
Training Setup

### ISSUE THREAD

**Issue by Soulaima-khammari** [boschresearch/MultiADS#9]
Hello everyone,

First of all, thank you for the great paper and the understandable code base.
I wanted to reproduce the results and train the model on the VisA dataset. When I looked into the `meta.json` file, I noticed that for training (`"train": {}`), only good data is used.

However, if I understood the paper correctly, the authors used anomalous images with their corresponding masks during training. Did I miss something, or are the meta.json files not intended to reflect the full training setup? Or is the training actually performed in an unsupervised manner?

Best regards,
Soulaima
